test/e2e_node/restart_test.go

```diff
@@ -150,6 +165,9 @@ var _ = SIGDescribe("Restart", framework.WithSerial(), framework.WithSlow(), fra
 
 	ginkgo.Context("Dbus", func() {
 		ginkgo.It("should continue to run pods after a restart", func(ctx context.Context) {
+			if isFedoraCoreOs() {
+				ginkgo.Skip("restarting dbus on fedora based os is not recommended")
+			}
```

Comment:
Just wondering how does it work on Ubuntu? If this is dangerous practice, we should consider removing this test.

As far as I know, after restarting `systemd` it's recommended to restart all systemd-* services, which is not done in this test. It could lead to hard to identify issues even on Ubuntu.

Reply:
Let's use @bart0sh PR actually then.
https://github.com/kubernetes/kubernetes/pull/120459
/close

Reply:
Thanks, I'll submit a PR for that.